preprocessors/openstreet/app/osm_service.py

```diff
@@ -0,0 +1,507 @@
+from typing import List
+import overpy
+from copy import deepcopy
+import haversine as hs
+import math
+
+
+def compute_query_bounding_box(distance_in_metres: float, lat: float, lon: float):
```

Comment:
Top level description should be provided for these functions with general information (e.g., how should these values be conceptually understood to relate to the resulting bounding box *without* stepping through the rest of the code).

Reply:
Top level description is now in the README file.  Also look at this: https://docs.google.com/document/d/1RN9tQKsTodWhX7qpr40jLGHbjzhyM4RrG0JYlSvcMrw/edit?usp=sharing